Clarify bools support (#405)
aio-libs/yarl: README.rst

The documentation is located at https://yarl.readthedocs.org

> Why boolean is not supported by URL query API?

Reviewer: ```suggestion
Why isn't boolean supported by the URL query API?
```

----------------------------------------------

There is no standard for boolean representation of boolean values.

Some systems prefer ``true``/``false``, others like ``yes``/``no``, ``on``/``off``,
``Y``/``N``, ``1``/``0``, etc.

> ``yarl`` cannot make an unambiguous decision how to serialize a ``bool`` values.

Reviewer: ```suggestion
``yarl`` cannot make an unambiguous decision on how to serialize ``bool`` values because it is specific to how the end-user's application is built and would be different for different apps.
```
